beets/__init__.py

```diff
@@ -27,12 +27,26 @@ class IncludeLazyConfig(confit.LazyConfig):
     """A version of Confit's LazyConfig that also merges in data from
     YAML files specified in an `include` setting.
     """
+    def __init__(self, *args, **kwargs):
+        super(IncludeLazyConfig, self).__init__(*args, **kwargs)
+
+        self._included_files = []
+
+    def user_config_paths(self):
+        """Points to a list of locations making up the user configuration.
+
+        The files may not exist.
+        """
+        return [self.user_config_path()] + self._included_files
```

Comment:
I'm not sure how to solve this, but if the file hasn't already been read then the included file list will be empty - should this be a note in the docstring or should this method actually call `read` if the config has not already been read?

Reply:
Stylistically, this method sounds like a getter, which may be better off served as an `@property`. As for your point, the latter makes sense, as I would expect the code to do whatever it needs to do to get my the `user_config_paths` when I call it.